app/files.go

```diff
@@ -44,14 +44,18 @@ import (
 	"os"
 	"os/signal"
 	"syscall"
+    "runtime/pprof"
+    "flag"
+    "runtime"
 
 	"github.com/TIBCOSoftware/flogo-lib/app"
 	"github.com/TIBCOSoftware/flogo-lib/engine"
 	"github.com/TIBCOSoftware/flogo-lib/logger"
 )
 
 var log = logger.GetLogger("main-engine")
-
+var cpuprofile = flag.String("cpuprofile", "", "write cpu profile to file")
```

Comment:
Perhaps a description like: 'Writes CPU profiling for the current process to the specified file' ?